docs/usage/enable_repo.md

```diff
@@ -3,8 +3,7 @@ title: "Enable a Repo"
 ---
 
 :::note
-You will need **Admin** access to the repo to be able to activate it in Vela. This is because you need **Admin** access to be able to add webhooks
-to the repo.
+You will need **Admin** access to the repo in your SCM (GitHub) to be able to activate it in Vela. This is because you need **Admin** access to be able to add webhooks to the repo.
```

Comment:
i though it wasn't super clear where you needed to be admin, hence the clarifier